### ISSUE TITLE
[NBug] Unable to parse object ID "fatal: not a git repository (or ...

### ISSUE THREAD

**Issue by zoghal** [gitextensions/gitextensions#7568]
:warning: The sections below must be filled in and this text must be removed or the issue will be closed.

## Error Details
```
System.FormatException: Unable to parse object ID "fatal: not a git repository (or any of the parent directories): .git".
   at GitUIPluginInterfaces.ObjectId.Parse(String s)
   at GitCommands.GitModule.GetCurrentCheckout()
   at GitUI.RevisionGridControl.ForceRefreshRevisions()
   at GitUI.FilterBranchHelper.ApplyBranchFilter(Boolean refresh)
   at System.Windows.Forms.ToolStripItem.RaiseEvent(Object key, EventArgs e)
   at System.Windows.Forms.Control.OnLeave(EventArgs e)
   at System.Windows.Forms.ContainerControl.UpdateFocusedControl()
```

## Environment
- Git Extensions 3.1.1.6049
- Build 2f872105b4eb42421d9fdafd68e90a9e27d20d96
- Git 2.24.1.windows.2
- Microsoft Windows NT 10.0.19035.0
- .NET Framework 4.8.4081.0
- DPI 96dpi (no scaling)

**Comment by pmiossec** [comment]
Must be fixed. Update to the last version.

**Comment by gerhardol** [comment]
Duplicate of #6512
Included in 3.2
